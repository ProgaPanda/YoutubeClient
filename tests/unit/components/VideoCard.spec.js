import { shallowMount } from '@vue/test-utils';
import VideoCard from '@/components/VideoCard.component.vue';

describe('VideoCard', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(VideoCard, {
      stubs: ['router-link'],
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
