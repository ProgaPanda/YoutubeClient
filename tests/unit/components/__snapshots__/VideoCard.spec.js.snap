// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoCard renders correctly 1`] = `
<router-link-stub
  to="[object Object]"
>
  <div
    class="video-card"
  >
    <div
      class="video-card__thumbnail"
    >
      <img
        alt="video-thumbnail"
      />
    </div>
     
    <div
      class="video-card__details"
    >
      <h4
        class="video-card__details__title"
      >
        
      </h4>
       
      <p
        class="video-card__details__meta"
      >
        
         
        <!---->
         
      
      </p>
       
      <p
        class="video-card__details__description"
      >
        
      </p>
    </div>
  </div>
</router-link-stub>
`;
